# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png
# Commit: 5ccaf8100f79
# Full Hash: 5ccaf8100f7912e55bb98cc1951d57df9c4dd511
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-04-26 16:04:35
# Regressor Bug: 1766017
# File Overlap Count: 1
# Description:
#   Bug 1766017 - Do a better job of clipping off-screen surface allocation sizes r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D144534
# ==============================================================================

diff -r 01d5f09a02bf -r 5ccaf8100f79 gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png
Binary file gfx/wr/wrench/reftests/filters/filter-drop-shadow-clip.png has changed